pages/contribute.md

```diff
@@ -6,44 +6,59 @@ main_nav_id: contribute
 
 # Contributing to Kotlin
 
-Kotlin is an Apache 2 OSS Project. The source code, tooling, documentation and even this web site is maintained on [GitHub](https://github.com/jetbrains/kotlin). While the majority 
-of the team works at JetBrains, there have been nearly a hundred external contributors to the Kotlin Project and we are always looking for more people. 
+Kotlin is an Apache 2 OSS Project. The source code, tooling, documentation and even this web site is maintained on [GitHub](https://github.com/jetbrains/kotlin). 
+While Kotlin is mostly developed by JetBrains, there are hundreds of external contributors to the Kotlin project&mdash;[become one of them](https://kotlinlang.org/community). 
```

Comment:
We shouldn't link the reader from here to the community section. There is no additional information for the contributors at the page and, we'll put the user on a closed loop.  
The paragraph also is introductory and recaps many different possible types of contribution, thus making it quite hard to point a reader to one place where they will become a contributor. In fact we will point the reader to many different places within further paragraphs of the guideline. That's why I think we don't need a CTA here at all.
My suggestion is we rephrase it somewhat like: "there are hundreds of external contributors to the Kotlin and you can find out how to become one of them in the following guidelines"

As an idea, at some point, we can provide a link to the list of contributors on GitHub https://github.com/JetBrains/kotlin/graphs/contributors, however, it will not represent those who contribute, for example, learning materials.

Reply:
Got your point! Removed CTA.